LIBSRC/SMB2_SHC/COM/smb2_shc.c

```diff
@@ -49,17 +49,22 @@ static const char IdentString[]=MENT_XSTR(MAK_REVISION);
 #define SHC_UPS_GET_OPCODE   0x05
 #define SHC_CONF_GET_OPCODE  0x06
 #define SHC_TEMP_SET_OPCODE  0x07
+#define SHC_GET_PERSISTENT_PWRBTN_OPCODE 0x08
```

Comment:
Quite long names, and not consistent to others. I would use SHC_PERS_PWRBTN_GET_OPCODE.
Or at last SHC_PERSISTENT_PWRBTN_GET_OPCODE (to be consistent).